frontend/openchat-worker/tsconfig.json

```diff
@@ -1,26 +1,9 @@
 {
+    "extends": "../tsconfig",
     "compilerOptions": {
-        "allowJs": true,
-        "checkJs": false,
-        "baseUrl": "./",
-        "declaration": true,
-        "experimentalDecorators": true,
-        "esModuleInterop": true,
-        "importsNotUsedAsValues": "error",
-        "lib": ["es2020", "WebWorker"],
-        "module": "es2020",
-        "moduleResolution": "node",
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
-        "outDir": "lib",
-        "paths": {},
-        "resolveJsonModule": true,
-        "rootDir": "./src",
-        "sourceMap": true,
-        "strict": true,
-        "target": "es2020",
-        "types": ["node"]
+        "module": "System",
```

Comment:
does this actually still work at runtime (as well as just building)?

Reply:
Fixed. Now it works using tsc and has the correct file name.